lms/djangoapps/grades/tests/test_scores.py

```diff
@@ -113,7 +113,9 @@ def _create_block(self, content_block_value):
             PersistedBlockValue(exists=True, raw_possible=5, weight=40, graded=True),
             ContentBlockValue(raw_possible=1, weight=20, explicit_graded=False),
             ExpectedResult(
-                raw_earned=None, raw_possible=None, weighted_earned=50, weighted_possible=100, weight=40, graded=True
+                raw_earned=None, raw_possible=None,
+                weighted_earned=50, weighted_possible=100,
+                weight=40, graded=True, attempted=True,
```

Comment:
(for all of these `ExpectedResult` blocks) Would these be clearer written as one item per line?